### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by IhostVlad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,157 +1,251 @@
...
+Allows to extend the standard resolve client and server configs.
 
+## **Environment variables**
 ### ROOT_DIR
+To change a root directory of an application, set the `ROOT_DIR` environment variable to the required value. For example, `export ROOT_DIR=/newurl`. After that the application will be available on [http://localhost:3000/newurl](http://localhost:3000/newurl). `ROOT_DIR` variable is available on the client side by `process.env.ROOT_DIR`.
 
-To change root directory of an application set the environment variable ROOT_DIR to needed value. For example `export ROOT_DIR=/newurl`. After that application will be available by `http://localhost:3000/newurl`. ROOT_DIR variable will be available on client side by `process.env.ROOT_DIR`.
-
-### Custom variables
+### Custom Environment Variables
```
Comment (VadimKhoroshiltsev) [inline README.md]: As I remember, we've decided to move ROOT_DIR section after "Custom variables".

Comment (IhostVlad) [inline README.md]: Environment variables section is consists of two parts: `ROOT_DIR` and `Custom user variables`. And what is the contradiction?

Comment (VadimKhoroshiltsev) [inline README.md]: Sorry, I wanted to say "Custom user variables" instead of "Custom variables"

Comment (VadimKhoroshiltsev) [inline README.md]: Because the ROOT_DIR is a special case of "Custom user variables"

**Pull request merged**